crates/ef-testing/src/evm_sequencer/sequencer/mod.rs

```diff
@@ -94,37 +96,49 @@ impl KakarotSequencer {
             vec![kakarot_address, evm_address.into()]
         };
 
-        let block_context = BlockContext {
-            block_info: BlockInfo {
-                block_number: BlockNumber(block_number),
-                block_timestamp: BlockTimestamp(block_timestamp),
-                sequencer_address: compute_starknet_address(
-                    &coinbase_address,
-                    environment.base_account_class_hash.0.into(),
-                    &coinbase_constructor_args,
-                )
-                .try_into()
-                .expect("Failed to convert to ContractAddress"),
-                vm_resource_fee_cost: Arc::new(VM_RESOURCES.clone()),
-                gas_prices: GasPrices {
-                    eth_l1_gas_price: 1,
-                    strk_l1_gas_price: 1,
-                    eth_l1_data_gas_price: 1,
-                    strk_l1_data_gas_price: 1,
-                },
-                use_kzg_da: false,
-                invoke_tx_max_n_steps: 50_000_000,
-                validate_max_n_steps: 50_000_000,
-                max_recursion_depth: 8192,
+        let block_info = BlockInfo {
+            block_number: BlockNumber(block_number),
+            block_timestamp: BlockTimestamp(block_timestamp),
+            sequencer_address: compute_starknet_address(
+                &coinbase_address,
+                environment.base_account_class_hash.0.into(),
+                &coinbase_constructor_args,
+            )
+            .try_into()
+            .expect("Failed to convert to ContractAddress"),
+            gas_prices: GasPrices {
+                eth_l1_gas_price: NonZeroU128::new(1).unwrap(),
+                strk_l1_gas_price: NonZeroU128::new(1).unwrap(),
+                eth_l1_data_gas_price: NonZeroU128::new(1).unwrap(),
+                strk_l1_data_gas_price: NonZeroU128::new(1).unwrap(),
             },
-            chain_info: ChainInfo {
-                chain_id: ChainId(String::from_utf8(chain_id.to_be_bytes().to_vec()).unwrap()),
-                fee_token_addresses: FeeTokenAddresses {
-                    eth_fee_token_address: *ETH_FEE_TOKEN_ADDRESS,
-                    strk_fee_token_address: *STRK_FEE_TOKEN_ADDRESS,
-                },
+            use_kzg_da: false,
+        };
+
+        let chain_info = ChainInfo {
+            chain_id: ChainId(String::from_utf8(chain_id.to_be_bytes().to_vec()).unwrap()),
+            fee_token_addresses: FeeTokenAddresses {
+                eth_fee_token_address: *ETH_FEE_TOKEN_ADDRESS,
+                strk_fee_token_address: *STRK_FEE_TOKEN_ADDRESS,
             },
         };
+
+        let versioned_constants: VersionedConstants =
+            serde_json::from_str(include_str!("./resources/versioned_constants.json"))
+                .expect("failed to parse versioned constants");
+
+        let bouncer_config = BouncerConfig::max();
```

Comment:
what is the `BouncerConfig`? What does setting it to `max()` mean for us?

Reply:
It's where we set the block capacity, didn't have that previously: https://github.com/starkware-libs/blockifier/blob/bad24f2543c848202f8fe2c833cd4149fef75d9f/crates/blockifier/src/bouncer.rs#L41

If the value is too small, we have transaction too large error. Just put the max to be fine with tests